bindings/bind_py_structure_manager.cc

```diff
@@ -794,12 +986,11 @@ namespace rascal {
     bind_update_unpacked<Manager_t>(manager);
   }
 
-  //! Bind the ClusterRef up to order 4 and from Layer 0 to 6
+  //! Bind the ClusterRef up to order 3 and from Layer 0 to 6
```

Comment:
out of curiosity, why remove the order 4?

Reply:
it's not used at the moment and probably won't be used in the near future so it just increases the compilation time for nothing.